src/index.html

```diff
@@ -1,13 +1,425 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Miami Condo Kings</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="images/favicon.png">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a
+              href="#menu"
+              class="icon icon--menu">
+            </a>
+
+            <a href="#home" class="logo">
+              <img
+                src="images/logo.png"
+                alt="Miami Condo Kings logo"
+                class="logo__image">
```

Comment:
Don't forget about style code

```suggestion
              <img
                src="images/logo.png"
                alt="Miami Condo Kings logo"
                class="logo__image"
              >
```